src/dashboard/Apps/AppsIndex.react.js

```diff
@@ -129,6 +131,15 @@ export default class AppsIndex extends React.Component {
       );
     }
     apps.sort((a, b) => a.createdAt > b.createdAt ? -1 : (a.createdAt < b.createdAt ? 1 : 0));
+    let upgradePrompt = null;
+    if (this.props.newFeaturesInLatestVersion.length > 0) {
+      let newFeaturesNodes = this.props.newFeaturesInLatestVersion.map(feature => <strong>
+        {feature}
+      </strong>);
+      upgradePrompt = <FlowFooter>
+        Upgrade to the latest version of Parse Dashboard to get access to: {joinWithFinal('', newFeaturesNodes, ', ', ' and ')}.
```

Comment:
can we make "latest version" link to: https://www.npmjs.com/package/parse-dashboard ?